Neos.Neos/Resources/Private/Fusion/Prototypes/BreadcrumbMenuItems.fusion

```diff
@@ -1,15 +1,19 @@
 prototype(Neos.Neos:BreadcrumbMenuItems) < prototype(Neos.Neos:MenuItems) {
-    itemCollection = ${q(documentNode).parents('[instanceof Neos.Neos:Document]').get()}
-    @process {
-        // Show always the current node, event when it is hidden in index
-        addCurrent = Neos.Fusion:Value {
-            currentItem = Neos.Neos:MenuItems {
-                renderHiddenInIndex = true
-                itemCollection = ${[documentNode]}
-            }
-            value = ${Array.concat(this.currentItem, value)}
-        }
+  // TODO Remove for Neos 9.0
+  @class = 'Neos\\Neos\\Fusion\\BreadcrumbMenuItemsImplementation'
+  includeSubItems = true
 
-        reverseOrder = ${Array.reverse(value)}
+  itemCollection = ${q(documentNode).parents('[instanceof Neos.Neos:Document]').get()}
+  @process {
+    // Show always the current node, event when it is hidden in index
+        addCurrent = Neos.Fusion:Value {
+      currentItem = Neos.Neos:MenuItems {
```

Comment:
Why are we even using MenuItems here in the first place? Can't we just append the current document?

Reply:
Because it's a menu, so it should have items, I guess. For B/C I'd leave that in place for now, refactor for 9.0.